docs/articles/roadmap/README.md

```diff
@@ -15,18 +15,16 @@ Please vote on the features you want us to implement first to help us prioritize
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Visual Regression Testing](https://github.com/DevExpress/testcafe/issues/1207)                   | Under Consideration
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Under Consideration
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Planned
```

Comment:
Can we say **In Progress** progress already?

Reply:
@kirovboris

Reply:
Sure